intel_extension_for_transformers/llm/runtime/graph/requirements.txt

```diff
@@ -1,5 +1,5 @@
 torch
-transformers
+transformers==4.33.1
```

Comment:
@a32543254 Seems that Mistral is only supported in transformers>=4.34. Using transformers==4.33.1 make "Auto classes" failed for Mistral. Should we update?